src/frontend/src/styles/main.css

```diff
@@ -254,6 +258,8 @@
   --rg-brand-bruised: var(--vc-brand-blue) 50%, var(--vc-brand-purple) 90%;
 
   /* reference tokens: sizes */
+  --rs-main-content-width: 40rem;
```

Comment:
reference tokens have to point to value tokens.

Reply:
Good point, done!